model/person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Please kindly implement the method to adding rentals as listed in the project requirement. More so the `rentals` variable supposed to be an `attr_accessor` because of the many to many relationship between **Person** and **Book** using the intermediate class **Rental**.